### DISCUSSION TITLE
visual debugging not working in jupyter note book

### DISCUSSION THREAD

**Discussion by sunday-will** [jsvine/pdfplumber#533] [Q&A] [answered]
hello!
**problem:**
I use notebook to try Visual debugging , but  image will not show, and qtconsole can show  image normally, and tried other computer have the same problem。have any suggestions.

![image](https://user-images.githubusercontent.com/42138516/140215251-055f40db-d8c8-46b3-a8bc-179594c6867a.png)

**environment:**

- pdfplumber version: 0.5.28
- Python version: 3.8.10
- jupyter notebook: 6.4.5
- OS:  win 7

**Comment by jsvine** [comment, marked as answer]
Instead of what you have in the screenshot, try this, with `im` as the final value in the cell, outside the `with` block:

```python
import pdfplumber

with pdfplumber.open('xxxx.pdf') as pdf:
  p = pdf.pages[0]
  im = p.to_image()

im
```
